core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -173,7 +173,32 @@ public static void startSinks(String metricsConfFile) {
   }
 
   /**
-   * Constructs the source name of metrics in this {@link MetricsSystem}.
+   * Constructs and returns the source name of metrics in this metrics system.
+   * <p>
+   * Sets {@code sourceKey} according to the value of {@link CommonUtils#PROCESS_TYPE}.
```

Comment:
I believe that it's unnecessary to reproduce the switch statement in Javadoc. Keep ip simple for the essence of the method, not the syntax.

Reply:
Fixed.